data/map.txt

```diff
@@ -25390,7 +25390,7 @@ planet "Into White"
 planet "Karek Fornati"
 	attributes korath efret
 	landscape land/desert1
-	description `Karek Fornati was once a farming world, before global temperatures began to rise and the overworked farmland slowly deteriorated into deserts. The Korath who remain have dedicated their lives to finding a way to keep the "sand seas" from swallowing up the last of the fertile land, but without much success, and there are regions near the equator where sandstorms rage continually and no living things can survive.`
+	description `Karek Fornati was once a farming world, before global temperatures began to rise and the overworked farmland slowly deteriorated into deserts. The Korath who remain have dedicated their lives to finding a way to keep the "sand seas" from swallowing up the last of the fertile land, but without much success, and there are regions near the equator where sandstorms rage continually and no living thing can survive.`
```

Comment:
I'd argue that either of these is grammatical, and they have slightly different implications. Saying "no living thing can survive" sounds more like it's saying that any living thing would be instantly annihilated, whereas, "No living things can survive" comes closer to my intended meaning: that the environment there is too hostile to sustain an ecosystem.

Reply:
The intended meaning sounds good to me.

I read "no living thing can survive" not as instant annihiliation but as "If you put a living thing in this environment, it would eventually die". That death would not necessarily occur in the blink of an eye, though; it might take some seconds or even days.

How about avoiding the word "thing", as in "...and neither animal nor plant can survive"? Or a slightly different take, "and bacterial colonies in sheltered caves are the only life form to be found"?